### DISCUSSION TITLE
WLANPI R4 does not show LLDP packets

### DISCUSSION THREAD

**Discussion by tstalzer** [WLAN-Pi/feedback#59] [WLAN Pi OS] [answered]
Hello
I have the latest version of wlanpi installed in a Raspberry Pi 4.
Network is connected to Eth0 directly to a switch which sends out LLDP packets (confirmed with other tools).
However WLANPI does not show the LLDP neighbor.
Do you have any ideas / tips on how to show the information?
Thanks
--Thomas

**Comment by jiribrejcha** [comment]
Sorry to hear, Thomas. Let's fix that. 

Can you please:
1. Connect your WLAN Pi to the switch via Ethernet
2. Wait for 60 seconds
3. SSH to your WLAN Pi and share these 2 files:
```
/tmp/lldpneightcpdump.cap
/tmp/lldpneigh.txt
```

4. Share output of this command:
```
date && cat /var/log/messages | grep "networkinfo script"
```

5. Share output of this command. It manually attempts to capture the LLDP frames coming from the switch into the WLAN Pi and display neighbour information in Shell.
```
sudo lldpneigh
```

Please review contents of the files before uploading. If you don't want to publicly share them here, feel free to e-mail me at jirka@jiribrejcha.net.

**Comment by jiribrejcha** [comment]
I forgot to ask, what switch model is it?

If you have a packet capture of the LLDP frame sent by the switch (Wireshark on a laptop should do the trick), that would be perfect.

**Comment by tstalzer** [comment, marked as answer]
Hello Jiri
Thanks for getting back to me.
The answer was actually much simpler.
Somehow the script could not write into 

```
/tmp/lldpneightcpdump.cap
/tmp/lldpneigh.txt
```

I deleted the files (needed root access - sudo -i) and then it was working. So just a rights thing.

Thanks for coming back to me.

--Thomas

**Reply by jiribrejcha** [reply]
Thanks for letting me know 👍🏻
